docs/source/guide/manage_data.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Manage data for your labeling project
+short: Manage data
+type: guide
+order: 401
+meta_title: Manage data for your labeling project
+meta_description: Manage, filter, and sort project data for your machine learning data science labeling project.
+---
+
+After you [set up your project](setup_project.html) and [labeling interface](setup.html) and [import your data](tasks.html), you can filter and sort your data to prepare it for [labeling](labeling.html).
```

Comment:
let's describe what Data Manager is 
1. screen shot
2. main panels
3. one row means = task
4. what else?